### ISSUE TITLE
PBI Documenter error message - Column [<myName>] is part of composite key

### ISSUE THREAD

**Issue by sherlockspreadsheets** [marclelijveld/External-Tools-Model-Documentation#43]
Getting an error message when running the PBI Documenter tool for one of our reports. This particular error message relates to the [new field parameter feature](https://powerbi.microsoft.com/en-my/blog/power-bi-may-2022-feature-summary/#post-19361-_Toc103022212), which we have implemented in the report. 

What is causing this error? How can I correct this so that PBI Documenter tool can be executed successfully? 

> Query (41, 87) Column [Dynamic VALUES] is part of composite key, but not all columns of the composite key are included in the expression or its dependent expression.
Error 0x0003: Error exporting vpax file.

![image](https://user-images.githubusercontent.com/39740261/184735241-124e1316-71e9-40f7-9ecc-15f53a4f030e.png)

---

### PBI Desktop 

Here is some information concerning the DAX table with documentation errors. 

```
'Dynamic VALUES' := 
{
    ("Disposal (groups)", NAMEOF('Disposal'[Disposal (groups)]), 0),
    ("Company (clean)", NAMEOF('Company'[Company (clean)]), 1),
    ("Operator (clean)", NAMEOF('Operator'[Operator (clean)]), 2),
    ("Hauler (clean)", NAMEOF('Hauler'[Hauler (clean)]), 3),
    ("Meter", NAMEOF('Meters Isometry'[Meter]), 4),
    ("Disposal (unified_name)", NAMEOF('Disposal'[Disposal (unified_name)]), 5),
    ("Company (unified_name)", NAMEOF('Company'[Company (company_unified_name)]), 6),
    ("Operator (unified_name)", NAMEOF('Operator'[Operator (company_unified_name)]), 7),
    ("Hauler (unified_name)", NAMEOF('Hauler'[Hauler (company_unified_name)]), 8),
    ("Business Entity", NAMEOF('Piped Trucked Volume'[business_entity]), 9),
    ("Gathering Type", NAMEOF('Piped Trucked Volume'[gathering_type]), 10),
    ("BWM Entity", NAMEOF('Disposal Attributes (calc field)'[BWM Entity]), 11),
    ("BWM Company", NAMEOF('Disposal Attributes (calc field)'[BWM Company]), 12),
    ("Tie In Type", NAMEOF('Meters Isometry'[Tie In Type]), 13)
}
```

![image](https://user-images.githubusercontent.com/39740261/184735015-217cc97a-a3b3-46b1-a8b0-be24ed60de57.png)

![image](https://user-images.githubusercontent.com/39740261/184735024-ec0e3fed-05c9-460e-9f2c-e09fe5e00d19.png)

![image](https://user-images.githubusercontent.com/39740261/184735043-cca28c3c-d962-4203-a650-aec3bd56b245.png)

**Comment by sherlockspreadsheets** [comment]
I just realized this is the same issue as: Issue https://github.com/marclelijveld/External-Tools-Model-Documentation/issues/39

**Comment by marclelijveld** [comment]
Closed issue as it is duplicate with #39
